compiler/annotation/expression/instructions/mod.rs

```diff
@@ -69,3 +70,11 @@ impl Error for ExpressionEvaluationError {
         }
     }
 }
+
+typedb_error!(
```

Comment:
WIP replacing this one here, I normally make a duplicate in the macro with suffix `2`, then delete the original and the `2` -- this leads to all the imports working everywhere automatically, then fix all the broken builds because the invokation is wrong!